recipes/jill/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+wget
```

Comment:
Is this really required, or did you only include it for pip check?

Reply:
Only for pip-check, since including it in the meta.yaml file made no difference. But I don't think that this works either.

Reply:
No this is just pip check not recognising it. Just remove it from here and list it under `run`– that should be enough.